functional_tests/prodigy/prodigy_integration.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+"""Test prodigy integration
+
+---
+id: 6.0.1
+name: prodigy integration test
+tag:
+  suite: nightly
+command:
+  timeout: 500
+plugin:
+  - wandb
+depend:
+  requirements:
+    - wandb
+    - spacy
+    - https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.0.0/en_core_web_md-3.0.0.tar.gz#egg=en_core_web_md
+  files:
+    - file: prodigy_test_resources.zip
+      source: https://raw.githubusercontent.com/wandb/wandb-testing/master/test_data/prodigy/prodigy_test_resources.zip
+assert:
+  - :wandb:runs_len: 1
+  - :wandb:runs[0][summary]:
+      image_segmentation[nrows]: 34
+      audio[nrows]: 3
+      nyt_tokens[nrows]: 200
+      quora_questions[nrows]: 100
+      already_labelled[nrows]: 373
+      products_html[nrows]: 200
+      unsplash_base64[nrows]: 207
+      unsplash_base64_corrupted[nrows]: 1
+      chinese_chars[nrows]: 30
+      unsplash[nrows]: 210
+      movies_ner[nrows]: 0
+      movies_tokens[nrows]: 200
+      nyt_classification[nrows]: 200
+      unsplash_food_options[nrows]: 37
+      videos[nrows]: 3
+      nyt_text[nrows]: 200
+      movies_cats[nrows]: 200
+      nyt_pos_tokens[nrows]: 246
+      compare_medical[nrows]: 300
+      nyt_ner[nrows]: 282
+      bio_tokens[nrows]: 129
+      bad_encoding[nrows]: 20
+      nyt_dep[nrows]: 112
```

Comment:
I am not sure what the expected result is that you are checking, i think you might want:
```
assert:
  - :wandb:runs_len: 1
  - :wandb:runs[0][summary][image_segmentation]: 34
  - :wandb:runs[0][summary][audio]: 3
```

but i dont know what [nrows] is trying to test so I might misunderstand.

Reply:
The main failure is because our mock server does not support artifacts very well, this will be added to yea soonish.
I will report back when it handles the base artifact cases.

Reply:
Will correct the asserts. I must've misinterpreted the format in the 0.6.1 PR earlier.

Reply:
It is the difference of doing:
```
summary_dict = {"key1": {"nested1": 11, "nested2": 12}, "key2": {"nested1": 21, "nested2": 22}}
# your asserts
assert summary_dict == {"key1": {"nested1": 11}, "key2": {"nested2": 11}}
# correct asserts
assert summary_dict["key1"]["nested1"] == 11
assert summary_dict["key2"]["nested1"] == 21
```

Reply:
It is much easier to write asserts when the test doesnt fail on an unsupported feature :)